src/app/components/nav-bar/nav-bar.component.ts

```diff
@@ -4,11 +4,13 @@ import { CommonModule } from '@angular/common';
 import { WindowContainerComponent } from '@app/components/window-container/window-container.component';
 import { UserPreferencesComponent } from '../user-preferences/user-preferences.component';
 import { DOCUMENT } from '@angular/common';
+import { WalletConnectComponent } from '@app/components/wallet-connect/wallet-connect.component';
+
 
 @Component({
     standalone: true,
     selector: 'app-nav-bar',
-    imports: [CommonModule, RouterModule, WindowContainerComponent, UserPreferencesComponent],
+    imports: [CommonModule, RouterModule, UserPreferencesComponent, WalletConnectComponent, WindowContainerComponent],
```

Comment:
```suggestion
imports: [
    CommonModule,
    RouterModule,
    UserPreferencesComponent,
    WalletConnectComponent,
    WindowContainerComponent,
],
```

Multiple imports from the same path should be placed one on each line according to best practices. It helps avoid file conflicts on PRs.